libs/dh/admin/feature-user-roles/src/lib/dh-user-roles.component.scss

```diff
@@ -33,6 +33,20 @@ watt-empty-state {
   padding: var(--watt-space-xl);
 }
 
+watt-expandable-card {
+  .mat-expansion-panel {
+    overflow: visible;
+  }
+
+  watt-expandable-card-title {
+    display: flex;
+    gap: var(--watt-space-s);
+    watt-icon {
+      color: var(--watt-color-primary-darker);
+    }
+  }
+}
+
 .loading-container {
   display: flex;
   justify-content: center;
```

Comment:
nitpick: some elements were removed entirely, which makes some of the styling unnecessary.